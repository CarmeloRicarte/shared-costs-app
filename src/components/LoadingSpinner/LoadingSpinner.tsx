import React from 'react';

const LoadingSpinner: React.FC = () => {
  return (
    <>
      <img src='Interwind-1s-200px.svg' alt='LoadingSpinner' />
    </>
  );
};

export default LoadingSpinner;
