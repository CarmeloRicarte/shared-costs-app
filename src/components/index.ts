export * from './CustomTable/CustomTable';
export * from './Modal';
