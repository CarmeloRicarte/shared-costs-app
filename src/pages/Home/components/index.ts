export * from './Balance';
export * from './CostsList';
export * from './CostsListItem';
