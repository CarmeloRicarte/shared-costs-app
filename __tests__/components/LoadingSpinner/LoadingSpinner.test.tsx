import { LoadingSpinner } from '@/components/LoadingSpinner';
import { render } from '@testing-library/react';

describe('LoadingSpinner tests', () => {
  test('should render the spinner', () => {
    const { container } = render(<LoadingSpinner />);
    expect(container).toMatchSnapshot();
  });
});
