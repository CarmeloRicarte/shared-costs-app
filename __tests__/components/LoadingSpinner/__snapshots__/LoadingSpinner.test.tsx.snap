// Vitest Snapshot v1

exports[`LoadingSpinner tests > should render the spinner 1`] = `
<div>
  <img
    alt="LoadingSpinner"
    src="Interwind-1s-200px.svg"
  />
</div>
`;
